CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
 Fix:  Fixed the bug about recovering ONTIMEINTERVAL subscriptions (Issue #693)
+Fix:  Fixed the crash in queryContext with a double geoscope (Issue #690)
```

Comment:
This text has to be modified also, the problem wasn't (as you detected) having a double geoscope, but that the polygon was not closed.